packages/repository/src/query.ts

```diff
@@ -221,6 +221,14 @@ export interface Filter<MT extends object = AnyObject> {
   include?: Inclusion[];
 }
 
+/**
+ * Filter without `where` property
+ */
+export type FilterExcludingWhere<MT extends object = AnyObject> = Omit<
+  Filter<MT>,
+  'where'
+>;
```

Comment:
I am not sure if it's worth introducing this type alias. IMO, `Omit` is a regular TypeScript tool that we are already using in other places (e.g. `Omit<Todo, 'id'>` when creating new model instances), what are you arguments against using `Omit<Filter<MyModel>, 'where'>`  instead of this new alias?

Reply:
Since the type is used for all generated CRUD controllers, I prefer to have a type alias to make it easier to understand.

Reply:
Fair enough.